scripts/linters/general_purpose_linter.py

```diff
@@ -166,6 +166,13 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
+    {
+        'regexp': re.compile(r'\bTestBed.get'),
+        'message': 'In unit tests, please use \'TestBed.inject\' instead of '
+                   '\'TestBed.get\'',
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
```

Comment:
We are moving all the checks to eslint, can you please write this as eslint plugin?